### DISCUSSION TITLE
Anilist ❗

### DISCUSSION THREAD

**Discussion by ghost** [lowlighter/metrics#1642] [Q&A]
Status                                                          │ complete
  MIME type                                                       │ image/svg+xml
  [
    {
      name: 'anilist',
      result: {
        error: {
          message: 'API error: 404 (Not Found.)',
          instance: {
            errors: [
              {
                message: 'Not Found.',
                status: 404,
                locations: [ { line: 2, column: 3 } ]
              }
            ],
            data: { User: null }
          }
        }
      }
    }
  ]

**Comment by EchoSingh** [comment]
ok so maybe you forget to add user try: -

```txt
          # Anime 
          plugin_anilist: yes
          plugin_anilist_user: your_user_name
          plugin_anilist_medias: anime
          plugin_anilist_sections: favorites, watching
          plugin_anilist_limit: 2
```
